pkg/workload/schemachange/operation_generator.go

```diff
@@ -3921,7 +3922,7 @@ func (og *operationGenerator) createFunction(ctx context.Context, tx pgx.Tx) (*o
 			return PickOne(og.params.rng, droppingEnums)
 		},
 		"ParamRefs": func() (string, error) {
-			refs, err := PickAtLeast(og.params.rng, 1, possibleParamReferences)
+			refs, err := PickBetween(og.params.rng, 1, optbuilder.MaxFuncParams-1, possibleParamReferences)
```

Comment:
i think we could make this avoid the `optbuilder` dependency, and just hardcode something like `20` here. it would be better for readability too.